lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=5
+SIDEKIQ_DB_POOL_SIZE=6
+SIDEKIQ_DB_REAPING_FREQUENCY=10
```

Comment:
- These new settings aren't documented in the README.
- It feels like `SIDEKIQ_DB_POOL_SIZE` should always be set to `SIDEKIQ_THREADS` + 1 (ie. not be configurable).
- OTOH it looks like `SIDEKIQ_DB_POOL_SIZE` isn't used anywhere( guessing it needs to be hooked up to the database configuration somehow?
- Same for `SIDEKIQ_DB_REAPING_FREQUENCY` as mentioned in another comment?